tests/acceptance/content.feature

```diff
@@ -0,0 +1,38 @@
+Feature: content
+  In order to manage content article in the web
+  As an owner
+  I need to create modify trash publish and Unpublish content article
+
+  Background:
+    Given Joomla CMS is installed
+    When Login into Joomla administrator with username "admin" and password "admin"
+    Then I see administrator dashboard
+
+  Scenario: Create an Article
+    Given There is a Add Content link
+    When I add the content with title "My_Article"
+    And I add content body "This is my first article"
```

Comment:
previous lines could be more readable like a table https://github.com/Codeception/Codeception/blob/master/docs/07-BDD.md#tables